spec/models/project_spec.rb

```diff
@@ -152,6 +153,10 @@
     describe "#establishment_sharepoint_link" do
       it { is_expected.to validate_presence_of :establishment_sharepoint_link }
     end
+
+    describe "#trust_sharepoint_link" do
+      it { is_expected.to validate_presence_of :establishment_sharepoint_link }
```

Comment:
Quick copy/paste snafu:

```suggestion
      it { is_expected.to validate_presence_of :trust_sharepoint_link }
```